ios/brave-ios/Sources/BraveWallet/Settings/DappsSettings.swift

```diff
@@ -253,6 +253,7 @@ private struct SiteConnectionDetailView: View {
         ForEach(siteConnection.connectedAddresses, id: \.self) { address in
           AccountView(
             address: address,
+            seed: address,  // TODO: Should be the same for blockie seed and the address in dapps for all networks including bitcoin. Bitcoin dapp site permission, we should store the AccountInfo.AccountId.UniqueKey.sha256. Might need to update the UI if the permission in bitcoin dapp will be handled differently.
```

Comment:
This TODO is very early, no Bitcoin DApp support yet, but it's good we checked early 😄! iOS `TODO`'s should have issue linked. I think either open iOS specific issue that links to https://github.com/brave/brave-browser/issues/34966 & include iOS issue # in this TODO comment, or drop this comment.

Reply:
can we somehow assert this view is eth sol only?

Reply:
@supermassive i dont think we need an assertion here since we only fetch permission status for `.dapps`
https://github.com/brave/brave-core/blob/dabc7d113156e4ed772dacfaf95a18b929ef1760/ios/brave-ios/Sources/BraveWallet/Crypto/Stores/ManageSiteConnectionsStore.swift#L51
And supported coin type is only for `.eth` and `.sol` which is stored here https://github.com/brave/brave-core/blob/dabc7d113156e4ed772dacfaf95a18b929ef1760/ios/brave-ios/Sources/BraveWallet/WalletConstants.swift#L100

Reply:
@supermassive i dont think we need an assertion here since we only fetch permission status for `.dapps`
https://github.com/brave/brave-core/blob/dabc7d113156e4ed772dacfaf95a18b929ef1760/ios/brave-ios/Sources/BraveWallet/Crypto/Stores/ManageSiteConnectionsStore.swift#L51
And supported coin type is only for `.eth` and `.sol` which are stored here https://github.com/brave/brave-core/blob/dabc7d113156e4ed772dacfaf95a18b929ef1760/ios/brave-ios/Sources/BraveWallet/WalletConstants.swift#L100